packages/rancher-operator/package.yaml

```diff
@@ -1,3 +1,3 @@
 url: https://github.com/rancher/rancher-operator/releases/download/v0.2.0-alpha02/rancher-operator-0.2.0-alpha02.tgz
 packageVersion: 01
-releaseCandidateVersion: 01
+releaseCandidateVersion: 02
```

Comment:
if packageVersion is 01 and rc version is 02, won't that make v0.2.0-alpha01-rc02 ? Should it be alpha02-rc01 ?

Reply:
You can run `make charts` to check

Reply:
Yes. Fixed.